### PR TITLE
Pja sk welcome user

### LINKED ISSUE #9: 7. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works
AC:

* The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

### PR DESCRIPTION
**Pull request by SaraSweetie**
## Description

For empty shopping lists, there is now text indicating the list is empty. The App home page also has more text to help users know what the app is.

## Related Issue

closes  #9 

## Acceptance Criteria

- The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

**Before:** The welcome page for new users.
Screen Shot 2020-04-27 at 1 31 19 PM

**Before:** After a user token is created the user is taken to an empty shopping list.
Screen Shot 2020-04-27 at 1 32 41 PM

### After
**After:** The welcome page for new users.
Screen Shot 2020-05-03 at 7 45 38 PM

**After:** When the Shopping List is empty.
Screen Shot 2020-05-03 at 7 48 14 PM

## Testing Steps / QA Criteria

1. Git pull the branch using `git pull origin pja-sk-welcome-user` and `git checkout pja-sk-welcome-user` to switch to the branch.
2. Run `npm install` to add dependencies, then `npm start` to run the app.
3. If you have a user token and items associated with that user token saved in the database, the app should perform as usual.
4. To view the welcome page, remove your user token. You can type `localStorage.removeItem();` in the browser's console. You can also remove a token under the Applications tab in the inspector.
5. After you remove your token, refresh the browser. You will see the welcome page. You will have the option to Join an existing list or to make a new empty one.
6. If you create a new empty list, then navigate to the `Shopping` view. You will see a message that your list is empty. Navigate to the `Add Item` view to add items. Go back to the shopping page to view the items you added.

### CODE REVIEW

**Comment by erostribe** [comment]
looks good to me, I think it's ready to merge and close.

**Review by espain16 — COMMENTED** [review]
Good Job ! I enjoy seeing everyones progress on this project.

**Review by skillitzimberg — APPROVED** [review]
Looks good. You got the link to the Add Item page! Nice!

File: src/components/shopping.js [hunk 1]
```diff
@@ -16,25 +16,35 @@ function Shopping(props) {
...
-          </div>
-        );
-      }}
-    />
-  );
+  if (props.list.length > 0) {
+    return (
+      <div>
+        <h1>Shopping List</h1>
+        <h2>***Click item to delete from cart***</h2>
```
Comment (stevelikesmusic) [inline src/components/shopping.js]:
```suggestion
        <h2>***Click item to delete from your list***</h2>
```

File: src/components/shopping.js [hunk 2]
```diff
@@ -16,25 +16,35 @@ function Shopping(props) {
...
-                  {item.name} / {item.next_purchase}
-                </li>
-              ))}
-            </ul>
-          </div>
-        );
-      }}
-    />
-  );
+  if (props.list.length > 0) {
```
Comment (stevelikesmusic) [inline src/components/shopping.js]: Not a blocker, just a React-y comment to think about later. You'll rarely see `if/else` returning JSX. More often you'll see the idiomatic React way to conditionally render is something like:

```jsx
return (
  {props.list.length > 0 
    ? <ul>{props.list.map(...renderListItem)}</ul>
	: <div><p>Your list is empty!</p></div>
  }
)
```

**Review by stevelikesmusic — APPROVED** [review]
Looks good 👏 Everything works like it should in the preview.

File: src/components/shopping.js (hunk 2 above)
Comment (SaraSweetie) [reply src/components/shopping.js]: this was resolved when merging

**Pull request merged**